src/ui/DynamicFacet/DynamicFacetValues/DynamicFacetRangeValueParser.ts

```diff
@@ -17,14 +17,10 @@ export class DynamicFacetRangeValueParser {
   }
 
   private parseDateFromRangeType(value: RangeType) {
-    switch (typeof value) {
-      case 'number':
-        return new Date(value as number);
-      case 'string':
-        return new Date(`${value}`);
-      default:
-        return value as Date;
-    }
+    const browserParsedDate = new Date(value as any);
+    return DateUtils.isValid(browserParsedDate)
+      ? browserParsedDate
+      : DateUtils.convertToStandardDate(value);
```

Comment:
Why not just keep and return this line? e.g.

```
private parseDateFromRangeType(value: RangeType) {
    return DateUtils.convertToStandardDate(value);
}
```

Reply:
Yes, I would not use standard browser date at all if possible, and the DateUtils module uses moment.js underneath, which is much more reliable.

Could we think about fallbacking to a default value that makes sense if event moment.js is unable to parse a date ? 
So that the component still "loads" somehow. That kind of pattern has been very helpful in the past, to try and recover from configuration error when possible.

With a warning logged in the console of course.

Reply:
In some cases/formats, Chrome can get dates that moment can't, but I would be down to use the default (which would just be a string of the value) and warning.

Reply:
But, I meant, returning something that will make it so the component still works in a sane default manner in the end ?

Reply:
Also, having different behaviour across browsers could result in a lot of maintenance cases being opened, with people saying they see one thing in Chrome and another in e.g. Firefox. I prefer to avoid such questions by returning the same thing consistently.